hack/util.sh

```diff
@@ -152,6 +152,44 @@ function start_os_server {
 	echo `date`
 }
 
+# start_os_api_server starts standalone OS's API. Node and Controllers need to be started
+# separately. Useful for testing controllers election.
+function start_os_api_server {
```

Comment:
This is currently specific to your extended test?  Let's keep this function in a file under your extended test.  I'm fine with the separate function, but I'd like to keep the main `util.sh` a "90% case" kind of file and this is pretty specific.